server/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -6638,8 +6638,6 @@ void assertLuceneOperations(InternalEngine engine, long expectedAppends, long ex
         assertThat(message, engine.getNumDocDeletes(), equalTo(expectedDeletes));
     }
 
-    // TODO Lucene 10 upgrade, we need to remove old IndexVersions for this to work
-    @AwaitsFix(bugUrl = "")
```

Comment:
Does this fail now?

Reply:
My main thinking is "with the 9x bump happened in main, why would we need to do anything special around index versions in lucene_snapshot_10?"